vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -0,0 +1,63 @@
+
+
+	
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means developers manage a central catalog of components and share it across an organization. Entando's distributed and composable development process allows you to quickly edit, test, analyze, and publish a singlular component or packaged business capability (PBC) repeatedly, without touching other parts of your application. This includes the ability to guide content, versioning, and metadata through the use of bundles. 
+
+To Curate and Share on Entando, see the docs below or follow a tutorial:
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+Business IT groups and developers Compose an Application with the Entando App Builder, a low-code composition UI. Assemble or update an application from the components already built in step one, adding templates, widgets, or full PBCs for functionality. The Entando Component Manager and Repository are built in to the App Builder, making the composable process flexible and agile by managing all your components there. 
+
+For more information on Composing an App, see the docs and tutorials below:
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder]<!--(../compose/app-builder.md)--> | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager]<!--(../compose/ecm-overview.md)--> | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume an Application 
+When the end-user interacts with an application, the Consume step provides an opportunity to monitor, analyze and enhance each building block. On Entando, the modular capability allows organizations to scale the app accordingly. This is the last step in the continuous process of composable apps, where feedback to creators and composers is built in. This means adding or editing web content, changing engine rules, or managing processes and useage are done independently and upgrades are responsive. 
```

Comment:
useage -> usage

Reply:
This paragraph was awkward in the source doc as well and isn't even technically accurate. We don't have a feature "where feedback to creators and composers is built in".  Maybe, "The 'Consume' step is the runtime phase of an Entando Application where an end-user (or consumer) interacts with the application to complete their specific task.  The modular design of the platform allows the application and services to be scaled according to that use.  In the same way the application can be continuously improved by adding or updating web content, changing engine roles, and deploying seamless upgrades to individual components.